File: C#-Part-1/Console Input Output/QuadraticEquation/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main()
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double discriminant = Math.Sqrt(Math.Pow(b, 2) - (4 * a * c));
        if (discriminant > 0)
        {
            double rootOne = (-b + discriminant) / (2 * a);
            double rootTwo = (-b - discriminant) / (2 * a);
            Console.WriteLine("{0:0.00}", Math.Min(rootOne, rootTwo));
            Console.WriteLine("{0:0.00}", Math.Max(rootOne, rootTwo));
        }
        else if (discriminant == 0)
        {
            double root = -b / (2 * a);
            Console.WriteLine("{0:0.00}", root);
        }
        else
        {
            Console.WriteLine("no real roots");
        }
    }
}

File: JavaScript Fundamentals/Conditional Statements/QuadraticEquation.js
function solve(args) {
    var a = +args[0],
        b = +args[1],
        c = +args[2];

    var discriminant = Math.sqrt((Math.pow(b, 2)) - (4 * a * c));
    if (discriminant > 0) {
        var xOne = (b + discriminant) / (2 * a);
        var xTwo = (b - discriminant) / (2 * a);

        if (xOne < xTwo) {
            console.log('x1=' + xOne.toFixed(2) + '; x2=' + xTwo.toFixed(2));
        }
        else{
            console.log('x1=' + xTwo.toFixed(2) + '; x2=' + xOne.toFixed(2));
        }
    }
    else if (discriminant === 0) {
        var root = b / (2 * a);
        console.log('x1=x2=' + root.toFixed(2));
    }
    else {
        console.log('no real roots');
    }
}
